admissions/consts.py

```diff
@@ -8,7 +8,8 @@ class Priority(models.TextChoices):
 
 
 class AdmissionStatus(models.TextChoices):
-    INITIALIZATION = ("initialization", "Initialization")
+    INITIALIZATION = ("configuration", "Configuration")
+    INTERVIEW_OVERVIEW = ("interview-overview", "Interview overview")
```

Comment:
`INTERVIEW_OVERVIEW` seems to not really server a purpose in the wizard. Delete this.